templates/default/filebeat-serving.yml.erb

```diff
@@ -0,0 +1,123 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: filebeat-config
+  namespace: hops-system
+  labels:
+    app: filebeat
+data:
+  filebeat.yml: |-
+    filebeat.autodiscover:
+      providers:
+        - type: kubernetes
+          node: ${NODE_NAME}
+          templates:
+            - condition.has_fields: ['kubernetes.labels.serving_hops_works/id']
```

Comment:
Isn't this picking up the logs for every container running? Including the control plane, Jupyter and KfServing control pods?

Reply:
I added the `serving.hops.works/*` labels for model serving. It will pick up the logs for every container inside a pod with a `serving.hops.works/id` label, which are kfserving inference service pods and the current serving deployments we have. We don't add these labels to Jupyter pods.